### PR TITLE
Fix Bug 1196573, update private browsing blurb on trust page

### PR DESCRIPTION
**Pull request by schalkneethling**
@alexgibson r?

@flodolo There are still an open question re the URL and explicit line breaks in the sub head and copy.

### CODE REVIEW

**Comment by alexgibson** [comment]
All these string changes will need l10n tags?

**Comment by alexgibson** [comment]
Also should we waffle this so it turns on once 42 is released?

**Comment by flodolo** [comment]
URL doesn't block us, since it's not part of the string. But yes, everything should be behind a tag, and we should display the DNT blurb for locales not ready.

**Comment by flodolo** [comment]
Actually, correction. We shouldn't display the DNT bit, just display the old Private Browsing text.

File: bedrock/firefox/templates/firefox/desktop/trust.html [hunk 1]
```diff
@@ -48,25 +48,28 @@
...
+              {{ _('Sometimes it’s nice to go undercover.') }}
+  -            {{ _('Turn this feature on to protect your browsing history from others.') }}
+            </p>
 
-          <a class="more" rel="external" data-interaction="outbound link" href="https://support.mozilla.org/kb/private-browsing-browse-web-without-saving-info">{{ _('Learn how to open a private window') }}</a>
-        </li>
+            <a class="more" rel="external" data-interaction="outbound link" href="https://support.mozilla.org/kb/private-browsing-browse-web-without-saving-info">{{ _('Learn how to open a private window') }}</a>
+          </li>
+        {% endif %}
         {% if l10n_has_tag('forget_button') %}
```
Comment (schalkneethling) [inline bedrock/firefox/templates/firefox/desktop/trust.html]: Do we have enough locales translated for the forget button or, is there still some that fallback to the lightbeam blurb? I removed the CSS for lightbeam and will need to add it back if the fallback is still displayed often. @flodolo ?

Comment (flodolo) [inline bedrock/firefox/templates/firefox/desktop/trust.html]: We have 9 locales that still don't have the strings translated, and we'd have to disable the page for them.

Having said that, I'm not against cleaning the template, but I don't think that's something we should do now, already too much going on.

File: bedrock/firefox/templates/firefox/desktop/trust.html [hunk 2]
```diff
@@ -48,25 +48,28 @@
...
+          <li id="privacy-browsing">
+            <h3>{{ _('Private Browsing') }}</h3>
 
-          <p>
-            {{ _('Sometimes it’s nice to go undercover.') }}
-            {{ _('Turn this feature on to protect your browsing history from others.') }}
-          </p>
+            <p>
+              {{ _('Sometimes it’s nice to go undercover.') }}
+  -            {{ _('Turn this feature on to protect your browsing history from others.') }}
```
Comment (flodolo) [inline bedrock/firefox/templates/firefox/desktop/trust.html]: Extra trailing minus

File: bedrock/firefox/templates/firefox/desktop/trust.html [hunk 3]
```diff
@@ -48,25 +48,28 @@
...
+            </p>
 
-          <a class="more" rel="external" data-interaction="outbound link" href="https://support.mozilla.org/kb/private-browsing-browse-web-without-saving-info">{{ _('Learn how to open a private window') }}</a>
-        </li>
```
Comment (flodolo) [inline bedrock/firefox/templates/firefox/desktop/trust.html]: This should be behind a tag as well.

**Comment by flodolo** [comment]
Not sure how final this is in terms of code, but alignment looks pretty bad outside of en-US
schermata 2015-10-01 alle 13 21 24

**Comment by schalkneethling** [comment]
Ah, I know what is wrong there.

File: bedrock/firefox/templates/firefox/desktop/trust.html (hunk 1 above)
Comment (schalkneethling) [inline bedrock/firefox/templates/firefox/desktop/trust.html]: Agreed

File: bedrock/firefox/templates/firefox/desktop/trust.html [hunk 4]
```diff
@@ -48,25 +48,28 @@
       </p>
 
       <ul class="feature-list">
-        <li id="privacy-dnt">
-          <h3>{{ _('Do Not Track') }}</h3>
+        {% if l10n_has_tag('tracking_protection') %}
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/desktop/trust.html]: We probably want to add a waffle switch here, so the new copy only shows if the locale has the l10n flag **and** the waffle switch is `true`, e.g. something like:

    {% if l10n_has_tag('tracking_protection') and waffle.switch('tracking_protection') %}

Comment (alexgibson) [inline bedrock/firefox/templates/firefox/desktop/trust.html]: Note: we seem to use a convention of using hyphens `-` in waffle switches looking at other pages, as opposed to underscores `_`.

File: bedrock/firefox/templates/firefox/desktop/trust.html [hunk 5]
```diff
@@ -48,25 +48,28 @@
       </p>
 
       <ul class="feature-list">
-        <li id="privacy-dnt">
-          <h3>{{ _('Do Not Track') }}</h3>
+        {% if l10n_has_tag('tracking-protection') %}
```
Comment (flodolo) [inline bedrock/firefox/templates/firefox/desktop/trust.html]: Can you switch to `tracking_protection` like the other page? We should not have issues anymore with hyphens, but still, underscore are safe and consistent ;-)

**Comment by schalkneethling** [comment]
Reade for code review @alexgibson r?

**Comment by schalkneethling** [comment]
@alexgibson waffle flag added.

**Comment by alexgibson** [comment]
This one has some centering issues also:

trust-alignment

**Comment by alexgibson** [comment]
Still alignment issues on this one at smaller viewports:

trust-alignment

**Comment by schalkneethling** [comment]
Oh man, I do not know how I do not pick these up :puzzled:

File: media/css/firefox/desktop/trust.less [hunk 6]
```diff
@@ -48,24 +48,38 @@
...
+            line-height: 1.3;
         }
-
-        &#privacy-forget {
-            background-position: -1310px top;
+        li {
+            float: none;
+            display: inline-block;
+            background: url(/media/img/firefox/desktop/trust/sprite-privacy.png) center top no-repeat;
+            margin-right: 30px;
```
Comment (alexgibson) [inline media/css/firefox/desktop/trust.less]: Is it intentional that this has a right margin of `30px`, while the `last-child` below has a left margin of `20px`?

File: media/css/firefox/desktop/trust.less [hunk 7]
```diff
@@ -48,24 +48,38 @@
...
+            padding-top: 180px;
+            vertical-align: top;
+
+            @media only screen and (max-width: @breakMobileLandscape) {
```
Comment (alexgibson) [inline media/css/firefox/desktop/trust.less]: Should this be `@breakTablet` seeing as it goes into single column here?

File: media/css/firefox/desktop/trust.less (hunk 6 above)
Comment (schalkneethling) [inline media/css/firefox/desktop/trust.less]: No, they should both be 20px

**Comment by alexgibson** [comment]
r+ looking good! :pineapple:

**Pull request merged**